sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/statsbeat/AttachStatsbeat.java

```diff
@@ -76,8 +76,8 @@ static String initResourceProviderId(
                 } else {
                     return UNKNOWN_RP_ID;
                 }
-            case RP_AKS: // TODO will update resourceProviderId when cluster_id becomes available from the
-                // AKS AzureMetadataService extension.
+            case RP_AKS:
+                return envVarFn.apply("AKS_ARM_NAMESPACE_ID ");
```

Comment:
```suggestion
                return envVarFn.apply("AKS_ARM_NAMESPACE_ID");
```

Reply:
oops, copy/paste and didn't realize an extra space.